docker/bookkeeper/Dockerfile

```diff
@@ -1,37 +1,51 @@
 #
-# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-FROM apache/bookkeeper:4.7.3
 
-ARG BK_VERSION=4.7.3
+#
+# Note: This file contains source code copied from the Apache BookKeeper
+#       project (http://bookkeeper.apache.org). Specifically, it contains
+#       source code copied from file `Dockerfile`, revisions `a14c8bf` and
+#       `a00f60d`:
+# - github.com/apache/bookkeeper/blob/branch-4.7/docker/Dockerfile
+# - github.com/apache/bookkeeper/blob/branch-4.9/docker/Dockerfile
+#
+
+FROM apache/bookkeeper:4.8.2
+
+ARG BK_VERSION=4.8.2
 ARG DISTRO_NAME=bookkeeper-all-${BK_VERSION}-bin
-ARG GPG_KEY=FD74402C
 
 RUN set -x \
-    && yum install -y iproute \
+    && yum install -y iproute wget \
     && cd /opt \
-    && curl -O "https://archive.apache.org/dist/bookkeeper/bookkeeper-${BK_VERSION}/${DISTRO_NAME}.tar.gz" \
-    && curl -O "https://archive.apache.org/dist/bookkeeper/bookkeeper-${BK_VERSION}/${DISTRO_NAME}.tar.gz.asc" \
-    && curl -O "https://archive.apache.org/dist/bookkeeper/bookkeeper-${BK_VERSION}/${DISTRO_NAME}.tar.gz.sha512" \
+    && wget -q "https://archive.apache.org/dist/bookkeeper/bookkeeper-${BK_VERSION}/${DISTRO_NAME}.tar.gz" \
```

Comment:
The change from `curl` to `wget` seems gratuitous in this PR. Is it necessary for the upgrade or just convenient?

Reply:
It is just convenient. The code in the `apache/bookkeeper` image [uses](https://github.com/apache/bookkeeper/blob/branch-4.8/docker/Dockerfile) `wget` as well, so it is consistent with what's used there and makes it easier to see the changes in our image on top of `Apache's`.